### ISSUE TITLE
"Colon expected at position" error

### ISSUE THREAD

**Issue by shurth** [josdejong/jsonrepair#99]
I saw that someone else reported an error, but it had to do with a left quote. Recently, I went to repair a validation schema and ran into the same, but it's with a regular quote inside of a single quoted string. It didn't matter if it was escaped (\\ or \) or not - it always results in "colon expected at XX". Here's a small example:

```javascript
{
  properties: {
    name: {
      type: "string",
      isRequired: true,
      pattern: '[^<>\\/\\?:;"\\[\\]\\{\\}\\|~`!@#$%\\^\\*=+]*'
    }
  }
}
```

If you remove the quote from the regex pattern, everything works as expected.

**Comment by josdejong** [comment]
Thanks for reporting. The issue here is coming from the backtick `` ` `` inside the string: `jsonrepair` assumes the backtick quote is the end of the string. This is to repair JSON coming from Word for example, where quotes and double quotes are often changed into round quotes. 

I'll look into it next week to see how we can better detect this and make repairing more strict.

**Comment by josdejong** [comment]
Fixed now in `v3.2.3`
